dissector/plugins/dummy/CMakeLists.txt

```diff
@@ -40,23 +40,16 @@ set(CLEAN_FILES
 set_source_files_properties(
         ${CLEAN_FILES}
         PROPERTIES
-        COMPILE_FLAGS ${WERROR_COMMON_FLAGS}
+        COMPILE_FLAGS "${WERROR_COMMON_FLAGS}"
```

Comment:
Quote `${WERROR_COMMON_FLAGS}`, or you'll get build error: `set_source_files_properties called with incorrect number of arguments`